ecr.go

```diff
@@ -0,0 +1,85 @@
+package main
+
+import (
+	"fmt"
+	"os"
+	"sort"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/awserr"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/ecr"
+)
+
+// for sorting setups
+// TODO: consider using global parameter 'query' to get sorted result directory from API
+// ref: https://docs.aws.amazon.com/cli/latest/reference/index.html
+type Images []ecr.ImageDetail // type alias to implement Len and Swap
+
+func (img Images) Len() int {
+	return len(img)
+}
+
+func (img Images) Swap(i, j int) {
+	img[i], img[j] = img[j], img[i]
+}
+
+type ByPushedAt struct {
+	Images
+}
+
+func (b ByPushedAt) Less(i, j int) bool {
+	ti := *b.Images[i].ImagePushedAt
+	tj := *b.Images[j].ImagePushedAt
+	return ti.Before(tj)
+}
+
+func retrieveFromECR(image string) ([]string, error) {
+	profile := os.Getenv("AWS_PROFILE")
+	if profile == "" {
+		fmt.Println("use aws default profile")
+	}
```

Comment:
`AWS_PROFILE` is a well-known variable in AWS CLI tools, so I think it doesn't have to notify using the default profile.